README.md

```diff
@@ -24,42 +24,47 @@ Based on [Officially Supported Databases](https://docs.djangoproject.com/en/3.0/
 
 ## Installation
 
+> ```
```

Comment:
What's this?

Reply:
@hslouyz That's not supposed to be there, removed now